src/types/zetkin.ts

```diff
@@ -196,4 +196,19 @@ export interface ZetkinTag {
     description: string;
 }
 
+export enum CUSTOM_FIELD_TYPE {
+    URL='url',
+    DATE='date',
+    TEXT='text',
+    JSON='json'
+}
+
+export interface ZetkinDataField {
```

Comment:
I think we should change the name of this type. This feature in Zetkin is communicated to users as "Custom fields" and internally in the API it's called "person fields". I think we should not introduce a third name ("data field").

Reply:
I've added this to #337